code/modules/antagonists/pirate/pirate_shuttle_equipment.dm

```diff
@@ -6,8 +6,11 @@
 	icon = 'icons/obj/machines/dominator.dmi'
 	icon_state = "dominator"
 	density = TRUE
+	/// Is the machine syphoning right now
```

Comment:
```suggestion
	/// Is the machine siphoning right now
```

i know it's a UK English vs American English thing but you spell it with an `i` everywhere else in this PR so let's be consistent